connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -305,7 +312,7 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
                 title: pageTitle,
                 url,
               },
-              `Document is empty or too big to be upserted. Skipping`
+              `Document is ${extracted.length > 0 ? "too big to be upserted" : "empty"}. Skipping`
```

Comment:
Not sure we to a lot of conditioning in the logger, but why not

Reply:
true, ill rollback